datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/DropDownViewHolderFactory.kt

```diff
@@ -103,7 +111,9 @@ internal object DropDownViewHolderFactory :
             )
             val selectedAnswer =
               questionnaireViewItem.answerOption
-                .firstOrNull { it.value.displayString(context) == selectedItem?.answerOptionString }
+                .firstOrNull {
+                  it.value.identifierString(context) == selectedItem?.answerIdentifierString
+                }
```

Comment:
can we just set selectedAnswer to this, I think it is doing the same thing is filtering for the id:
```suggestion
  val selectedAnswer =
              questionnaireViewItem.answers.singleOrNull()?.value
```

Reply:
This fails initially when `questionnaireVIewItem.answers` is empty, essentially it never gets to select new